src/components/learner-credit-management/invite-modal/FileUpload.jsx

```diff
@@ -6,9 +6,10 @@ import {
 import { InsertDriveFile } from '@edx/paragon/icons';
 
 import { formatBytes } from '../../MultipleFileInputField/utils';
+import InviteModalInputFeedback from './InviteModalInputFeedback';
 import { MAX_EMAIL_ENTRY_LIMIT } from '../cards/data';
```

Comment:
nit: Don't need to import `MAX_EMAIL_ENTRY_LIMIT` anymore
```suggestion
import InviteModalInputFeedback from './InviteModalInputFeedback';
```